### PR TITLE
[344] Surface last login date for support users

### PR DESCRIPTION
**Pull request by avinhurry**
## Context

For security, GDPR and Fraud purposes, we should regularly audit the users that have access to the Support Console in Production, removing those that no longer require it. 

To enable a more formal, criterial approach it would be useful to have the “last login date” for each each user on the “Support Users” page to allow this to be reviewed and actioned by a non-dev member of the team.

## Changes proposed in this pull request

- Add last signed in date/time column to the support users index page.
- Sort the users list by last signed in date/time, with the oldest date/time at the top of the list.
- If the user has never logged in, show static text describing this (as seen in the screenshot below).

## Screenshot

image

## Guidance to review

- Check out the page, add new users, login with different users and check that the order is correct (or look at the screenshot above).

## Things to check

- [x] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [x] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [x] API release notes have been updated if necessary
- [x] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [x] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)
- [x] Inform data insights team due to database changes
- [x] Make sure all information from the Trello card is in here
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally
- [x] Add PR link to Trello card

### CODE REVIEW

**Comment by avinhurry** [comment]
[Add the last login date for each person in "Support Users" page on Support Console](https://trello.com/c/SSsOwCW1/344-add-the-last-login-date-for-each-person-in-support-users-page-on-support-console)

File: app/controllers/support_interface/support_users_controller.rb [hunk 1]
```diff
@@ -1,7 +1,8 @@
 module SupportInterface
   class SupportUsersController < SupportInterfaceController
     def index
-      @support_users = params[:removed] == 'true' ? SupportUser.discarded : SupportUser.kept
+      @unsorted_support_users = params[:removed] == 'true' ? SupportUser.discarded : SupportUser.kept
```
Comment (elceebee) [inline app/controllers/support_interface/support_users_controller.rb:4]:
```suggestion
      unsorted_support_users = params[:removed] == 'true' ? SupportUser.discarded : SupportUser.kept
```

does this need to be an instance variable? Do we use it in the index controller, or do we always use `@support_users`?

File: spec/system/support_interface/managing_support_users_spec.rb [hunk 2]
```diff
@@ -7,7 +7,7 @@
     given_i_am_a_support_user
 
     when_i_visit_the_support_console
-    and_i_click_the_manange_support_users_link
+    and_i_click_the_manage_support_users_link
```
Comment (elceebee) [inline spec/system/support_interface/managing_support_users_spec.rb:10]: 👍 thank you!

**Review by elceebee — APPROVED** [review]
Just one small thing, not a blocker. I've tested on the review app and it works as expected. Nice one!

File: app/controllers/support_interface/support_users_controller.rb (hunk 1 above)
Comment (avinhurry) [reply app/controllers/support_interface/support_users_controller.rb]: Nope you are right, I was supposed to do this but forgot, thank you. 

https://github.com/DFE-Digital/apply-for-teacher-training/pull/9963/commits/aff0cfda3b33d05565c5d083e6476d378c307650

**Pull request merged**